### PR TITLE
Add customCheckboxSource render prop

### PR DESCRIPTION
**Pull request by HichamELBSI**
Hi, 

A quick fix to https://github.com/tableflip/react-native-select-multiple/issues/7

Regards

### CODE REVIEW

**Comment by HichamELBSI** [comment]
Hi @alanshaw, did i mess something else ?

File: src/SelectMultiple.js [hunk 1]
```diff
@@ -132,27 +133,33 @@ export default class SelectMultiple extends Component {
...
+      } else {
+        rowStyle = mergeStyles(styles.row, rowStyle)
+        checkboxStyle = mergeStyles(styles.checkbox, checkboxStyle)
+        labelStyle = mergeStyles(styles.label, labelStyle)
+      }
     }
 
     return (
       <TouchableWithoutFeedback onPress={() => this.onRowPress(row)}>
         <View style={rowStyle}>
```
Comment (alanshaw) [inline src/SelectMultiple.js]: No merged row styles if `renderCustomCheckboxSource`?

File: src/SelectMultiple.js [hunk 2]
```diff
@@ -132,27 +133,33 @@ export default class SelectMultiple extends Component {
     } = this.props
 
     const {
+      renderCustomCheckboxSource,
```
Comment (alanshaw) [inline src/SelectMultiple.js]: rename to just `renderCheckbox`?

**Review by alanshaw — CHANGES_REQUESTED** [review]

**Comment by HichamELBSI** [comment]
Hey @alanshaw, what do you think about the last commit ? 😅 sorry for spamming you man

**Comment by alanshaw** [comment]
@HichamELBSI no worries sorry for the delay, can you please rebase?

**Comment by alanshaw** [comment]
ping @HichamELBSI

**Comment by Jacklol** [comment]
what problem width custom checkbox? i need this too. can I help?

**Pull request closed without merging**